challenges/objects-arrays.js

```diff
@@ -6,29 +6,51 @@
   object name, diet, weight, length, period
 */
 
-// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
-
+const tRex = {
+  name:'tyrannosaurus',
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous'
+  
+};
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
+ const  steggy = {
+  name: 'stegosaurus',
+  diet: 'herbivorous',
+  weight: '2000kg',
+  length: '9m',
+  period: 'Late CJurassic'
+ };
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
-
-// Using your dinosaur objects, log answers to these questions:
+ const vRaptor = {
+  name: 'velociraptor',
+  diet: 'carnivorous',
+  weight: '15kg',
+  length: '1.8m',
+  period: 'Late Jurassic'
+ };
+// 
 
 // How much did tyrannosaurus weigh?
-console.log();
+console.log(tRex.weight);
 
 // What was the diet of a velociraptor?
-console.log();
+console.log(vRaptor.diet);
 
 // How long was a stegosaurus?
-console.log();
+console.log(steggy.weight);
 
 // What time period did tyrannosaurus live in?
-console.log();
+console.log(tRex.period)
 
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
-console.log();
+ this.speak = function roar() {
+  return "RAWERSRARARWERSARARARRRR!"
+}
```

Comment:
So `this` is binded to the window here. You would want to target complete this challenge as such:

```js
tRex.speak = function roar() {
  return "RAWERSRARARWERSARARARRRR!"
}
```